src/test/lib/TestConnection.cpp

```diff
@@ -295,6 +295,7 @@ TestConnection::SetQuicVersion(
     uint32_t value
     )
 {
+#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
```

Comment:
It's unfortunate that we broke this test, which used to work with the old param. But oh well.